config/environments/development.rb

```diff
@@ -23,7 +23,9 @@
   }
   config.action_mailer.asset_host = Figaro.env.mailer_domain_name
   config.action_mailer.smtp_settings = { address: ENV['SMTP_HOST'] || 'localhost', port: 1025 }
-  config.action_mailer.default_options = { from: Figaro.env.email_from }
+  config.action_mailer.default_options = {
+    from: "#{Figaro.env.email_from_display_name} <#{Figaro.env.email_from}>",
```

Comment:
Can we use the `mail` gem or equivalent to generate these? I know that these values come from our configs, and they're relatively safe/controller, but I always worry a bit when interpolating email addresses like this. That way, if we ever choose to style the display name with `<`, it will display properly? 

```suggestion
    from: Mail::Address.new.tap do |mail|
      mail.address = Figaro.env.email_from
      mail.display_mail = Figaro.env.email_from_display_name
    end.to_s,
 ```

(and also for the other environments, too)

Reply:
And considering that this code is the same in all environments, maybe we should just move it to `application.rb`?

Reply:
Great suggestion to use the `mail` gem. And I agree, I'll move everything into `application.rb`.